examples/simple.go

```diff
@@ -33,45 +32,61 @@ func main() {
 		frostdb.NewTableConfig(schema),
 	)
 
-	// Create values to insert into the database these first rows havel dynamic label names of 'firstname' and 'surname'
-	buf, _ := schema.NewBuffer(map[string][]string{
-		"names": {"firstname", "surname"},
-	})
+	type FirstLast struct {
+		FirstName string
+		Surname   string
+	}
 
-	// firstname:Frederic surname:Brancz 100
-	_, _ = buf.WriteRows([]parquet.Row{{
-		parquet.ValueOf("Frederic").Level(0, 1, 0),
-		parquet.ValueOf("Brancz").Level(0, 1, 1),
-		parquet.ValueOf(100).Level(0, 0, 2),
-	}})
+	type Simple struct {
+		Names FirstLast
+		Value int64
+	}
+	// Create values to insert into the database these first rows havel dynamic label names of 'firstname' and 'surname'
+	frederic := Simple{
+		Names: FirstLast{
+			FirstName: "Frederic",
+			Surname:   "Brancz",
+		},
+		Value: 100,
+	}
 
-	// firstname:Thor surname:Hansen 10
-	_, _ = buf.WriteRows([]parquet.Row{{
-		parquet.ValueOf("Thor").Level(0, 1, 0),
-		parquet.ValueOf("Hansen").Level(0, 1, 1),
-		parquet.ValueOf(10).Level(0, 0, 2),
-	}})
-	_, _ = table.InsertBuffer(context.Background(), buf)
+	thor := Simple{
+		Names: FirstLast{
+			FirstName: "Thor",
+			Surname:   "Hansen",
+		},
+		Value: 99,
+	}
+	_, _ = table.Write(context.Background(), frederic, thor)
 
 	// Now we can insert rows that have middle names into our dynamic column
-	buf, _ = schema.NewBuffer(map[string][]string{
-		"names": {"firstname", "middlename", "surname"},
-	})
-	// firstname:Matthias middlename:Oliver surname:Loibl 1
-	_, _ = buf.WriteRows([]parquet.Row{{
-		parquet.ValueOf("Matthias").Level(0, 1, 0),
-		parquet.ValueOf("Oliver").Level(0, 1, 1),
-		parquet.ValueOf("Loibl").Level(0, 1, 2),
-		parquet.ValueOf(1).Level(0, 0, 3),
-	}})
-	_, _ = table.InsertBuffer(context.Background(), buf)
+	matthias := struct {
+		Names struct {
+			FirstName  string
+			MiddleName string
+			Surname    string
+		}
+		Value int64
+	}{
+		Names: struct {
+			FirstName  string
+			MiddleName string
+			Surname    string
+		}{
+			FirstName:  "Matthias",
+			MiddleName: "Oliver",
```

Comment:
lol, first time seeing this example 😄 
```suggestion
			MiddleName: "Oliver Rainer",
```
😝